mappings/net/minecraft/command/arguments/TestClassArgumentType.mapping

```diff
@@ -1,4 +1,6 @@
 CLASS net/minecraft/class_4526 net/minecraft/command/arguments/TestClassArgumentType
+	FIELD field_20580 examples Ljava/util/Collection;
```

Comment:
if this is `static final` per yarn rule it should be
```suggestion
	FIELD field_20580 EXAMPLES Ljava/util/Collection;
```
even though the field may not be effectively constant.